docs/self-managed/platform-deployment/ingress-setup.md

```diff
@@ -0,0 +1,164 @@
+---
+id: ingress-setup
+title: "Combined and separated Ingress setup"
+description: "Camunda Platform 8 Self-Managed combined and separated Ingress setup"
+---
+
+Camunda Platform 8 Self-Managed has multiple web applications and gRPC services. Both can be accessed externally using Ingress. There are two ways to do this:
+
+1. **Combined setup:** In this setup, there are two Ingress objects: one Ingress object for all Camunda Platform web applications using a single domain (each application has a sub-path e.g. `camunda.example.com/operate`,) and another which uses gRPC protocol for Zeebe Gateway e.g. `zeebe.camunda.example.com`.
+2. **Separated setup:** In this setup, each component has its own Ingress/host e.g. `operate.camunda.example.com`, `optimize.camunda.example.com`, `zeebe.camunda.example.com`, etc.
+
+There are no significant differences between the two setups. Rather, they both offer flexibility for different workflows.
+
+:::note
+Camunda Platform Helm chart doesn't manage or deploy Ingress controllers, it only deploys Ingress objects. Hence, this Ingress setup will not work without Ingress controller running in your cluster.
+:::
+
+## Preparation
+
+- An Ingress controller should be deployed in advance. The examples below use `Nginx` Ingress controller, but any Ingress controller could be used by setting `ingress.className`.
```

Comment:
I adjusted to "The examples below use `Nginx` Ingress controller" -- is this what you meant?

Reply:
yes exactly :+1: